dll/win32/browseui/shellbrowser.cpp

```diff
@@ -802,13 +805,33 @@ HRESULT CShellBrowser::Initialize()
     return S_OK;
 }
 
+HRESULT CShellBrowser::ApplyBrowserDefaultFolderSettings(IShellView *pvs)
+{
+    HRESULT hr;
+    if (pvs)
+    {
+        m_settings.Save();
+        SBFOLDERSETTINGS &sbfs = m_deffoldersettings, defsbfs;
```

Comment:
```suggestion
        SBFOLDERSETTINGS& sbfs = m_deffoldersettings, defsbfs;
```

Reply:
Stop suggesting this on my PRs, the style guide does not say where it goes and you fail to answer when I ask you about where it goes for `TYPEX& foo, &bar`.